adapter/src/main/java/com/twilio/kudu/sql/KuduPrepareImpl.java

```diff
@@ -325,13 +330,20 @@ public void executeDdl(Context context, SqlNode node) {
               cubeColumnSchemas.add(columnSchemaBuilder.build());
             }
             // if node is a column from the original table use the same column schema
-            else {
+            else if (kuduTable.getSchema().hasColumn(sqlnode.toString())) {
               ColumnSchema colSchema = kuduTable.getSchema().getColumn(sqlnode.toString());
               cubeColumnSchemas.add(colSchema);
             }
           }
         }
 
+        // add the count_records column
+        org.apache.kudu.Common.DataType dataType = Common.DataType.INT64;
```

Comment:
This feels like a Twilio specific thing. Does this belong in this repo?

Reply:
Probable we should add it in readme and keep it. As removing this involves fixes to the datasets and other places (which whimsically sometimes do have the measure and sometimes dont)

Reply:
I have removed this. I modified this so that you can specify an alias that determines the column name used to store the aggregated value. For eg.
```
SUM(column_val) -> SUM_COLUMN_VAL
SUM(column_val as "sum_column_val" -> sum_column_val
COUNT(*) as "count_records" -> count_records
```